ntbs-integration-tests/PatientPageTests.cs

```diff
@@ -0,0 +1,48 @@
+using System.Collections.Generic;
+using System.Net;
+using System.Threading.Tasks;
+using ntbs_integration_tests.Helpers;
+using ntbs_service;
+using ntbs_service.Models.Validations;
+using Xunit;
+
+namespace ntbs_integration_tests
+{
+    public class PatientPageTests : TestRunnerBase
+    {

```

Comment:
won't complete this model as part of this ticket, just made a start on it

Reply:
Sure thing - can you add a code comment to highlight that this is not a complete one?